__tests__/AzureSqlAction.test.ts

```diff
@@ -7,8 +7,6 @@ import DotnetUtils from '../src/DotnetUtils';
 import SqlConnectionConfig from '../src/SqlConnectionConfig';
 import Constants from '../src/Constants';
 
-jest.mock('fs');
```

Comment:
With the update tests were failing with same error as https://github.com/actions/toolkit/issues/1075

Removing the mock seems to resolve the issue. Every method tested here that calls into `fs` are mocked anyway